common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -282,7 +289,7 @@ div.video {
         }
       }
 
-      ul.vcr {
+      div.vcr {
```

Comment:
> NIT: If we can avoid adding the "div" element style requirement here that would be great. Additionally, it wasn't clear to me from this what vcr stood for. It might be helpful to use more verbose class labeling here.

 done

> I just noticed that .vcr seems to be a wrapper class around .vidtime. If that is its primary use, I could consider using ".wrapper-video-duration" instead of "vcr" and ".video-duration" instead of .vidtime

It might require a lot of changes (js, unit tests, bok-choy, etc) and it's not a part of this PR.